pkg/dryrun/dryrun_utils.go

```diff
@@ -317,3 +318,22 @@ func PathExists(path string) bool {
 
 	return err == nil
 }
+
+func addColorToDiff(bareDiff string, noColor bool) string {
+	lines := strings.Split(bareDiff, "\n")
+	for i, line := range lines {
+		if strings.HasPrefix(line, "-") && !strings.HasPrefix(line, "---") {
```

Comment:
(Optional) I'd personally be fine if the header were also colored...

```suggestion
		if strings.HasPrefix(line, "-") {
```

Reply:
<img width="863" alt="Screenshot 2025-04-11 at 12 24 33 PM" src="https://github.com/user-attachments/assets/08021706-9267-4454-b6f6-8001ceb62e18" />
git diff doesn't add color to "---" so. If we add the color? same color red and green?

Reply:
Oh, we can leave it then--I didn't recall that git handled it this way also. 🙂

Reply:
I like it as-is